packages/vanilla-renderers/src/complex/categorization/CategorizationList.tsx

```diff
@@ -37,7 +36,7 @@ const getCategoryClassName = (
 ): string => (selectedCategory === category ? 'selected' : '');
 
 export interface CategorizationProps {
-  categorization: Categorization;
+  filteredCategories: [Category];
```

Comment:
I think this type is incorrect: `[Category]` is an array with exactly one `Category` element. `Category[]` is a normal array of `Category` elements.

Furthermore, I think handing in only the filtered categories is not enough. Child categorizations are lost with this approach. I think the type should probably be `(Category | Categorization)[]` to allow both. In fact it effectively gets an array of this type at the moment because `Categorization`s are not filtered out in `CategorizationRenderer` ;)